composer.json

```diff
@@ -70,6 +71,7 @@
         "symfony/phpunit-bridge": "^4.3"
     },
     "suggest": {
+        "nelmio/api-doc-bundle": "^2.0",
```

Comment:
Please remove the version here, the suggest section expects a description

Reply:
done